src/cloud/components/RateLimitAlertContent.tsx

```diff
@@ -75,6 +62,32 @@ const UpgradeMessage: FC<UpgradeMessageProps> = ({limitText, link, type}) => {
       important metrics.
     </span>
   )
+
+  return isFlagEnabled('credit250Experiment') ? (
+    <GoogleOptimizeExperiment
+      experimentID={CREDIT_250_EXPERIMENT_ID}
+      original={original}
+      variants={[
+        <span className="upgrade-message" key="1">
+          You hit the{' '}
+          <a
+            href={link}
+            className="rate-alert--docs-link"
+            target="_blank"
+            rel="noreferrer"
+          >
```

Comment:
there's a component, [`SafeBlankLink`](https://github.com/influxdata/ui/blob/master/src/utils/SafeBlankLink.tsx) to handle this for you.

[here](https://github.com/influxdata/ui/blob/master/src/homepageExperience/components/steps/nodejs/WriteData.tsx#L108-L113) is an example of it being used